screengrab/lib/screengrab/runner.rb

```diff
@@ -69,6 +69,8 @@ def run
 
       open_screenshots_summary(device_type_dir_name)
```

Comment:
Should we skip opening screenshots in picture browser (since reports are implemented)?